### PR TITLE
k9s: add page

### PR DESCRIPTION
**Pull request by xRuiAlves**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):** `0.27.4`

### CODE REVIEW

File: pages/common/k9s.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# k9s
+
+> CLI to view and manage Kubernetes clusters.
```
Comment (kbdharun) [inline pages/common/k9s.md:3]:
```suggestion
> View and manage Kubernetes clusters.
```
Ref. https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#descriptions

File: pages/common/k9s.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+> More information: <https://k9scli.io/>.
```
Comment (kbdharun) [inline pages/common/k9s.md:4]:
```suggestion
> More information: <https://k9scli.io/topics/commands/>.
```
I would suggest directly linking to the commands page.

File: pages/common/k9s.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+- Manage a cluster in "read-only" mode (disabling all commands that may cause modifications):
+
+`k9s --readonly --cluster {{cluster_name}}`
+
+- Manage a cluster using a given kubernetes namespace:
+
+`k9s --namespace {{kubernetes_namespace}} --cluster {{cluster_name}}`
+
+- Manage a cluster launching k9s in the "pod" view and enabling debug logging:
```
Comment (kbdharun) [inline pages/common/k9s.md:18]:
```suggestion
- Manage a cluster launching k9s in the pod view and enable debug logging:
```

File: pages/common/k9s.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
+# k9s
+
+> CLI to view and manage Kubernetes clusters.
+> More information: <https://k9scli.io/>.
+
+- Manage a cluster using a kubeconfig context:
+
+`k9s --context {{kubeconfig_context_name}}`
+
+- Manage a cluster in "read-only" mode (disabling all commands that may cause modifications):
```
Comment (kbdharun) [inline pages/common/k9s.md:10]:
```suggestion
- Manage a cluster in read-only mode (disabling all commands that may cause modifications):
```

**Review by kbdharun — COMMENTED** [review]
Thanks for your contribution. I have some suggestions for this page.

File: pages/common/k9s.md (hunk 1 above)
Comment (xRuiAlves) [reply pages/common/k9s.md:3]: Thanks for the suggestion!

File: pages/common/k9s.md (hunk 3 above)
Comment (xRuiAlves) [reply pages/common/k9s.md:18]: k9s: update command style description

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Review by blueskyson — APPROVED** [review]

**Pull request merged**